oaff/testing/Dockerfile

```
@@ -1,10 +1,15 @@
 FROM oaff
 
-RUN apk add --no-cache --update postgresql-client
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    postgresql-client-11 \
+  && rm -rf /var/lib/apt/lists/*
 
-COPY requirements-test.txt /requirements-test.txt
-RUN pip install -r /requirements-test.txt
+WORKDIR /opt/$NAME
+COPY requirements-test.txt requirements-test.txt
+RUN pip install -r requirements-test.txt --no-cache-dir 
 
+WORKDIR /
 RUN mkdir -p /oaff/testing
 COPY data /oaff/testing/data
```

If you implement my suggestion above, `WORKDIR` will already be set to `/opt/ogc-api-fast-features`
```suggestion
WORKDIR /opt/$NAME
COPY requirements-test.txt ./
RUN pip install --no-cache-dir -r requirements-test.txt

RUN mkdir -p oaff/testing
COPY data oaff/testing/data
```